extensions/imagesvg/src/common/Types.ts

```diff
@@ -12,16 +12,40 @@ export interface ImageSvgStyle extends RXTypes.ViewStyle {
 }
 export declare type ImageSvgStyleRuleSet = RXTypes.StyleRuleSet<ImageSvgStyle>;
 
-export interface SvgCommonProps {
-    key?: string | number;
+type NumberProp = string | number;
+
+export type FillRule = 'evenodd' | 'nonzero';
+export type Units = 'userSpaceOnUse' | 'objectBoundingBox';
+
+export type Linecap = 'butt' | 'square' | 'round';
+export type Linejoin = 'miter' | 'bevel' | 'round';
+
+export interface FillProps {
+    fillColor?: string; // Different from react-native-svg and web impl -- they just use "fill"
+    fillOpacity?: NumberProp;
+    fillRule?: FillRule;
+}
+  
+export interface StrokeProps {
+    strokeColor?: string;   // Different from react-native-svg and web impl -- they just use "stroke"
+    strokeWidth?: NumberProp;
+    strokeOpacity?: NumberProp;
+    // These typings are incompatible between web and react-native-svg.  Requires further understanding...
+    // strokeDasharray?: ReadonlyArray<number> | string,
+    strokeDashoffset?: NumberProp;
+    strokeLinecap?: Linecap;
+    strokeLinejoin?: Linejoin;
+    strokeMiterlimit?: NumberProp;
+}
+
+export interface ImageSvgProps extends RXTypes.CommonStyledProps<ImageSvgStyleRuleSet> {
+    // Wacky subset of available props that only does anything on web..?
```

Comment:
Not sure what you mean by "wacky subset...that only does anything on web". These were implemented previously for both web and native. Is the issue that they're defined for images rather than paths?